src/test/java/org/minions/devfund/abel/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,123 @@
+package org.minions.devfund.abel.moviedatabase;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for Movie Database Unit Tests.
+ */
+public class MovieDatabaseTest {
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "NewMovie";
+        movieDatabase.addMovie(newMovie, new String[]{"Actor1", "Actor2"});
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor2" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor1" 12 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)